spec/factories/users.rb

```diff
@@ -5,10 +5,11 @@
   factory :user do
     email { Faker::Internet.email }
     password { 'swapsea' }
-    club.name { 'Swapsea SLSC' }
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
     gender { 'male' }
+
+    association :club, factory: :club
```

Comment:
This can be shortened to just `association :club` given the factory is the same name as the association. I've worked on code bases where we always defined the factory but happy to :fire: it here.

Reply:
Sometimes more is more.